src/SprykerEco/Yves/Payone/PayoneDependencyProvider.php

```diff
@@ -73,4 +77,18 @@ protected function addQuoteClient(Container $container): Container
 
         return $container;
     }
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addStore(Container $container): Container
+    {
+        $container->set(static::STORE_CONFIG, function () {
+            return Store::getInstance();
```

Comment:
* Please do not use `Store::getInstance()` it's forbidden.
* Use `\Spryker\Client\Store\StoreClient::getCurrentStore()`.

Reply:
++

Reply:
++